middleware/redirects.ts

```diff
@@ -1,27 +1,49 @@
 export default function ({ redirect, route }): void {
   const { urlPrefix } = usePrefix()
 
-  if (route.path === '/drops') {
-    return redirect('/ahk/drops')
-  }
+  let redirectValue
 
-  if (route.path.startsWith(`/${urlPrefix.value}`)) {
-    if (route.path.endsWith('collections')) {
-      return redirect(`/${urlPrefix.value}/explore/collectibles`)
-    }
-    if (route.path.endsWith('gallery')) {
-      return redirect(`/${urlPrefix.value}/explore/items`)
-    }
-  }
+  console.log(route.path)
 
-  if (route.path.includes('/stmn/')) {
-    return redirect(window.location.href.replace('/stmn/', '/ahk/'))
-  }
+  const paths = [
+    {
+      cond: (val) => val === '/drops',
+      replaceValue: () => '/ahk/drops',
```

Comment:
does replaceValue need to be a function? 
I believe it should work even if it is not. 
maybe 
```
{
      cond: (val) => val === '/drops',
      url: '/ahk/drops',
}
```

Reply:
I can change to simple string and check if it's a callable function.

Reply:
@pbkompasz  that is not what i meant. Sorry let me explain, all replaceValue field doesn't need to be a function i believe. Is there a reason why these are functions?